pom.xml

```diff
@@ -147,8 +150,6 @@
     <junit.version>4.13.2</junit.version>
     <hamcrest.version>1.3</hamcrest.version>
     <junit5.version>5.10.2</junit5.version>
-    <assertj-core.version>3.25.3</assertj-core.version>
-    <awaitility.version>4.2.0</awaitility.version>
```

Comment:
I'd suggest to not move these in the same PR. If there's a need to put these in alphabetical order, it's better to do that separately and do it consistently. Currently the testing related libraries are kept together and that's why assertj-core and awaitility versions are after junit5 version.

Reply:
@lhotari Okay